### ISSUE TITLE
[FlightDetails.Json]: 'View' button under 'FlightDetails' card is getting cropped in normal view.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#93]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

**Repro Steps:**
1. Turn on the Talkback.
2. Navigate to the Visual.
3. Navigate to the sample card.
4. Using swipe gesture, swipe left to right in the card.
5. Navigate to FlightDetails.Json button by using swipe gesture and double tap to it.
6. FlightDetails adaptive card will be appeared.
7. Navigate to FlightDetails card by using swipe gesture and observe the issue.

**Actual Result:**
While navigating the card in normal view, 'View' button is getting cropped and not visible properly.

**Expected Result:**
In normal view, 'View' button should be visible properly and it should not cropped.

**User Impact:**
It will impact the low vision user as they will not be able to see the entire button text if the button text will be cropped in normal mode.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

!['View' button under 'FlightDetails' card is getting cropped in normal view](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/036caad0-e5d2-4c1f-b27b-ebfea6c143fe)

### Card JSON

```JSON
NA
```
